src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -986,6 +988,28 @@ void ClassesDCmd::execute(DCmdSource source, TRAPS) {
   VMThread::execute(&vmop);
 }
 
+#if INCLUDE_CDS
```

Comment:
A simple style nit-pick:  Error!
The JVM doesn't really dramatize problems like that.  It tells you there was an error, and calmly describes the problem after a colon. "Error: " 8-)

(There are some "Error!" exclamations in java.xml and one has got into jdk.jfr, but they are unusual.)

You can tell me that nit-picking is NOT hyphenated if you like.